src/host/services/IMUService.cpp

```diff
@@ -71,29 +97,36 @@ void IMUService::loop() {
   update.setSource(SKSource::sourceForKBoxSensor(SKSourceInputKBoxIMU));
 
   // In the SignalK Specification
-  //  roll:   Vessel roll, +ve is list to starboard
+  //  roll:   Vessel roll, +ve is list, heels to starboard
   //  pitch:  Pitch, +ve is bow up
   switch (_config.mounting) {
-    case VerticalPortHull:
+    case verticalRightSideToBow:
       _roll = SKDegToRad(eulerAngles.z());
-      _pitch = SKDegToRad(eulerAngles.y());
+      _pitch = SKDegToRad(eulerAngles.y())*(-1);
       _heading = SKDegToRad(fmod(eulerAngles.x() + 270, 360));
     break;
-    case VerticalStbHull:
+    case verticalLeftSideToBow:
       _roll = SKDegToRad(eulerAngles.z())*(-1);
-      _pitch = SKDegToRad(eulerAngles.y())*(-1);
-      _heading = SKDegToRad(eulerAngles.x());
+      _pitch = SKDegToRad(eulerAngles.y());
+      _heading = SKDegToRad(fmod(eulerAngles.x() + 90, 360));
     break;
-    case VerticalTopToBow:
+    case verticalBottomToBow:
+    case horizontalBottomToBow:
       _roll = SKDegToRad(eulerAngles.y());
       _pitch = SKDegToRad(eulerAngles.z());
       _heading = SKDegToRad(fmod(eulerAngles.x() + 180, 360));
     break;
-    case HorizontalLeftSideToBow:
-      _roll = SKDegToRad(eulerAngles.y());
-      _pitch = SKDegToRad(eulerAngles.z()) * (-1);
+    case verticalTopToBow:
```

Comment:
Same here. Can you let me know which one you have tested and which ones are untested?

Reply:
@sarfata good that you asked for more tests ;-)
Found and corrected a bug in inverting angles

Tested all 8 position variants successfully now

(You could do a final test with one of the positions with KBox, just to be sure I got it right. 
Not 100% sure with the BNO chip positioning in KBox)


Just wondering why it doesn´t do any new checks....